CHANGES.rst

```diff
@@ -13,6 +13,18 @@ Changes
   The meta data us useful, for example, for selecting tests with the
   nose attribute mechanism.
 
+- Added ``doctestcase.doctestfiles``
+
+  - Define multiple doctest files at once.
+
+  - Automatically assign test class members.  So rather than::
+
+      test_foo = doctestcase.doctestfile('foo.txt')
+
+    You can use::
+
+      doctestcase.doctestfiles('foo.txt')
```

Comment:
"It's not obvious that calling something in class scope defines new members.", except it's documented: "Automatically assign test class members.".

I agree with the suspicion of the magic.  But I think the DRY benefits outweigh the documented magic:

    doctestcase.doctestfiles('foo.txt')

if less repetitious than:

    test_foo = doctestcase.doctestfile('foo.txt')

which is especially important with names longer than foo.